locksmith/__tests__/operations/userOperations.test.ts

```diff
@@ -0,0 +1,86 @@
+import UserOperations from '../../src/operations/userOperations'
+
+const models = require('../../src/models')
+let User: any = models.User
+let UserReference: any = models.UserReference
+
+describe('User creation', () => {
+  let userCreationDetails = {
+    emailAddress: 'USER@EXAMPLE.COM',
```

Comment:
want to put a quick // saying we use upercase to ensure that data is stored in lower case?